flagging_site/data/database.py

```diff
@@ -105,17 +107,22 @@ def update_database():
     model_outs.to_sql('model_outputs', **options)
 
 
+@dataclass
 class boathouses(db.Model):
```

Comment:
small but important: turn this into `Boathouses` with a capital letter. This is one of the more important things to adhere to with PEP 8: all classes should start with capital letters. The exceptions to this are very rare, and only apply for function-like classes (can start with lowercase) e.g. context managers are allowed to start with lowercase.

Reply:
I have capitalized class Boathouses.